src/main/java/io/vertx/core/http/impl/ClientHandler.java

```diff
@@ -107,14 +106,14 @@ protected void handleMessage(ClientConnection conn, ContextImpl context, Channel
         case BINARY:
         case CONTINUATION:
         case TEXT:
+        case PONG:
```

Comment:
Should we also handle ping here and expose a `pingHandler` method in `WebSocketBase`? We could default it to `buf -> { writePong(buf); }` to provide default behavior, but some users may want to log metrics (for example in a load test client).

Reply:
no, the case is different for websocket because there is no payload, so no need to receive specific data, in general the application is not interested to receive ping (as it would anyway response with a pong). However it is interested to send ping to check if the connection is still alive.